cate/ds/esa_cci_odp.py

```diff
@@ -1265,19 +1268,29 @@ def reporthook(block_number, read_size, total_file_size):
                     sub_monitor_msg = "file %d of %d" % (file_number, len(outdated_file_list))
                     with child_monitor.starting(sub_monitor_msg, file_size):
                         actual_url = url[_ODP_PROTOCOL_HTTP]
-                        _LOG.info(f"Downloading {actual_url} to {dataset_file}")
+
+                        if http_redirect:
+                            urlpart = actual_url.split('/', 2)[2]
```

Comment:
Please use `urllib.parse.urlparse()` to get the actual protocol. This will also make sure the URL is valid.